orcoursetrion/tests/test_github.py

```diff
@@ -337,6 +341,132 @@ def test_lib_put_team_membership_fail(self):
         ):
             git_hub.put_team(self.ORG, self.TEST_TEAM, True, [])
 
+    def test_lib_copy_repo(self):
+        """Verify that we can do a single commit, single branch copy of a
+        repo."""
+
+        from orcoursetrion import config
+
+        commit_1 = 'hello'
+        commit_2 = 'world'
+        branch = 'foo'
+
+        SRC_REPO = 'Thing1'
+        DST_REPO = 'Thing2'
+
+        prefix = 'orc_git_test'
+
+        # Create directories to use.
+
+        # Note: If you are getting weird errors about ``[No such file
+        # or directory]: getcwd()`` when this test fails and you are
+        # debugging, disable the cleanup here. Even with the cleanup
+        # of changing the working directory below, pytest seems to run
+        # and the working directory is the tmp directories we nuked in
+        # cleanup.
```

Comment:
Interesting. What this says looks accurate: It's deleting the `tmp_dir_dst` during cleanup, and it looks like you'd always be in `DST_REPO_CLONE` at that point (a sub directory of `tmp_dir_dst`. Did you ever actually encounter this failing, or did you just suspect that it might fail?